apps/base-docs/tutorials/docs/3_farcaster-frames-gating-and-redirects.md

```diff
@@ -186,12 +144,12 @@ You can now add outbound links to buttons. To do this with [OnchainKit], simply
 
 ### Redirect Button
 
-The third button contains a `Redirect to pictures ↗`. The way it works is a little tricky. Start with the `buttons` defined in the original frame in `page.tsx`. The third button is a `post_redirect`:
+You can also do a redirect with a button. The example has one on the second frame - `Dog pictures ↗`. The way it works is a little tricky. Start with the `buttons` defined in the response frame in `app/frame/route.tsx`. The third button is a `post_redirect`:
 
```

Comment:
I wasn't sure if you are telling me to add a redirect button or suggesting. Can you clarify?

Reply:
Should the user have 4 or 3 buttons?

```javascript
buttons: [
    {
      label: 'Story time',
    },
    {
      action: 'tx',
      label: 'Send Base Sepolia',
      target: `${NEXT_PUBLIC_URL}/api/tx`,
      postUrl: `${NEXT_PUBLIC_URL}/api/tx-success`,
    },
    {
      action: 'link',
      label: 'Link to Google',
      target: 'https://www.google.com',
    },
    {
      action: 'post_redirect',
      label: 'Dog pictures',
    },
  ],
```